oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/TokenType.java

```diff
@@ -15,17 +15,18 @@
  */
 package org.springframework.security.oauth2.server.authorization;
 
-import org.springframework.util.Assert;
-
 import java.io.Serializable;
 
+import org.springframework.util.Assert;
+
 /**
  * @author Joe Grandja
  */
 public final class TokenType implements Serializable {
 	private static final long serialVersionUID = Version.SERIAL_VERSION_UID;
 	public static final TokenType ACCESS_TOKEN = new TokenType("access_token");
 	public static final TokenType REFRESH_TOKEN = new TokenType("refresh_token");
+	public static final TokenType ID_TOKEN = new TokenType("id_token");
```

Comment:
Maybe we want to create a task to add some additional tests after we add proper OIDC and _id_token_ support